### PR TITLE
Add recipe for Sync.com

### LINKED ISSUE #175: [Recipe Request] Add Sync cloud storage
Sync is very respected company for offering by default zero knowledge cloud service even with their free account which is limited to 5 GB - see:
https://www.sync.com/

### PR DESCRIPTION
**Pull request by marcolussetti**
Adds support for sync.com, addressing #175 

No notifications needed/appropriate for this service.

### CODE REVIEW

**Comment by marcolussetti** [comment]
Looks like icon must be square or CI will fail. Fixed, will update docs to make that clear as well.

**Comment by marcolussetti** [comment]
(Sorry for the close/reopen. Misclicked.)

**Comment by marcolussetti** [comment]
Looks like CI still thinks it's not square. My svg-fu is not the greatest so I'll have to revisit this later today with a bit of help I have lined up.

File: recipes/sync/icon.svg [hunk 1]
```diff
@@ -0,0 +1 @@
...25.76-12.11l.54-.58a41.49,41.49,0,0,0,11.74-28.21v-.87A2.27,2.27,0,0,0,171.86,86.61Zm-20.74,21.32a29,29,0,0,1-19,8.29A45.23,45.23,0,0,0,130.22,106l.45-.66a17.86,17.86,0,0,0,13.19-5.69h0A20.1,20.1,0,0,0,150.46,86a15.9,15.9,0,0,0-4.4-12.08h0a25.23,25.23,0,0,0-14-5.72h0A17.71,17.71,0,0,0,118,73.71h-.41c-.38.37-.92.79-1.25,1.16l4.57,3.52-20.37,6.85L102,63.79l5.31,4.15a24.43,24.43,0,1,0-35.71,6.8,63.14,63.14,0,0,0,12.44,6.35c9,3.61,18.33,5,25.8,11.7a35.34,35.34,0,0,1-44.8,54.59,34.41,34.41,0,0,1-12.44-32.23,21.8,21.8,0,0,1-7.8,1.28v.83h0v-1a33.17,33.17,0,0,1-21.57-8.29H23a28.55,28.55,0,0,1-8.75-20.74,29,29,0,0,1,8.3-20.29v.25A28.55,28.55,0,0,1,41,58.86a46.7,46.7,0,0,0,1.78,9.87h0a17.71,17.71,0,0,0-12,5.72h0a17.51,17.51,0,0,0-5.76,12.07,18.76,18.76,0,0,0,4.68,13.11h0a19,19,0,0,0,12.82,5.73,18.57,18.57,0,0,0,13.52-5.19h.25a11.76,11.76,0,0,0,1.08-1l-4.81-3.36,20.24-6.59-1.33,21.36L66,106.39a24.47,24.47,0,1,0,35.55-6.22A32.23,32.23,0,0,0,98,97.93h0C87.37,92.41,74.76,90.76,64.8,83.54a35.59,35.59,0,0,1,43.64-56.21,36,36,0,0,1,14.23,32.19A32.42,32.42,0,0,1,132,58.15h0a29.07,29.07,0,0,1,19.79,8.3h-.42a29,29,0,0,1,8.3,20.28,28.29,28.29,0,0,1-8.59,20.74Z" transform="translate(-1.46 -7.5)"/></svg>
```
Comment (kytwb) [inline recipes/sync/icon.svg:1]:
```suggestion
<svg id="Layer_1" data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 170.4 170.4"><defs><style>.cls-1{fill:#2dabe2;}.cls-2{fill:#fff;}</style></defs><title>icon</title><path class="cls-1" d="M152.62,60.52l-.42-.38a38.2,38.2,0,0,0-22.81-10.37A43.81,43.81,0,0,0,86.66,15.14,43.2,43.2,0,0,0,52,32.06a44.35,44.35,0,0,0-8.09,17.8A37.92,37.92,0,0,0,21.12,60.39l-.45.42A37.68,37.68,0,0,0,9.39,86.9v1.49a37.82,37.82,0,0,0,10.74,25.39l.58.62A37.9,37.9,0,0,0,44,125.35a43.68,43.68,0,0,0,85.37,0,38.08,38.08,0,0,0,23.35-11l.5-.54a37.6,37.6,0,0,0,10.66-25.59V86.77A38,38,0,0,0,152.62,60.52Z" transform="translate(-1.46 -7.5)"/><path class="cls-2" d="M171.86,86.61a41.74,41.74,0,0,0-12.44-29l-.46-.41a41.88,41.88,0,0,0-25.18-11.45A48.25,48.25,0,0,0,86.66,7.51a47.59,47.59,0,0,0-38.2,18.66,48.8,48.8,0,0,0-8.92,19.54A41.93,41.93,0,0,0,14.41,57.32l-.5.46A41.46,41.46,0,0,0,1.46,86.57v1.66a41.47,41.47,0,0,0,11.87,28l.62.67A41.83,41.83,0,0,0,39.62,129a48.16,48.16,0,0,0,94.16,0,41.9,41.9,0,0,0,25.76-12.11l.54-.58a41.49,41.49,0,0,0,11.74-28.21v-.87A2.27,2.27,0,0,0,171.86,86.61Zm-20.74,21.32a29,29,0,0,1-19,8.29A45.23,45.23,0,0,0,130.22,106l.45-.66a17.86,17.86,0,0,0,13.19-5.69h0A20.1,20.1,0,0,0,150.46,86a15.9,15.9,0,0,0-4.4-12.08h0a25.23,25.23,0,0,0-14-5.72h0A17.71,17.71,0,0,0,118,73.71h-.41c-.38.37-.92.79-1.25,1.16l4.57,3.52-20.37,6.85L102,63.79l5.31,4.15a24.43,24.43,0,1,0-35.71,6.8,63.14,63.14,0,0,0,12.44,6.35c9,3.61,18.33,5,25.8,11.7a35.34,35.34,0,0,1-44.8,54.59,34.41,34.41,0,0,1-12.44-32.23,21.8,21.8,0,0,1-7.8,1.28v.83h0v-1a33.17,33.17,0,0,1-21.57-8.29H23a28.55,28.55,0,0,1-8.75-20.74,29,29,0,0,1,8.3-20.29v.25A28.55,28.55,0,0,1,41,58.86a46.7,46.7,0,0,0,1.78,9.87h0a17.71,17.71,0,0,0-12,5.72h0a17.51,17.51,0,0,0-5.76,12.07,18.76,18.76,0,0,0,4.68,13.11h0a19,19,0,0,0,12.82,5.73,18.57,18.57,0,0,0,13.52-5.19h.25a11.76,11.76,0,0,0,1.08-1l-4.81-3.36,20.24-6.59-1.33,21.36L66,106.39a24.47,24.47,0,1,0,35.55-6.22A32.23,32.23,0,0,0,98,97.93h0C87.37,92.41,74.76,90.76,64.8,83.54a35.59,35.59,0,0,1,43.64-56.21,36,36,0,0,1,14.23,32.19A32.42,32.42,0,0,1,132,58.15h0a29.07,29.07,0,0,1,19.79,8.3h-.42a29,29,0,0,1,8.3,20.28,28.29,28.29,0,0,1-8.59,20.74Z" transform="translate(-1.46 -7.5)"/></svg>
```

This should fix it, even though I am not sure that's the right way to do it.

**Comment by kytwb** [comment]
The PNG also needs to be 1024x1024:

```
⚠️ Couldn't package "sync": Recipe PNG icon dimensions should be 1024x1024
```

File: recipes/sync/icon.svg (hunk 1 above)
Comment (marcolussetti) [reply recipes/sync/icon.svg]: Yeah that makes sense -- thanks!

**Comment by marcolussetti** [comment]
Fixed the PNG & SVG, I hope. (I thought I had the PNG at 1024x1024 but got mixed up on the versions).

**Comment by kytwb** [comment]
@marcolussetti welcome, thanks for your contributions!

**Pull request merged**